config/environments/development.rb

```diff
@@ -67,4 +67,20 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.delivery_method = :sendmail
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_options = { from: Rails.application.credentials.gmail[:email] }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    domain: 'gmail.com',
+    user_name: Rails.application.credentials.gmail[:email],
+    password: Rails.application.credentials.gmail[:password],
+    authentication: 'plain',
+    enable_starttls_auto: true
+  }
```

Comment:
- [ ] [OPT] I suggest that you don't use a real email delivery in your development environment as this can cause extra complexity to your dev cycle. Instead, I encourage you to use something like the [`letter_opener`](https://github.com/ryanb/letter_opener) gem to "simulate" the real mail interaction! 🤓 🔧